### ISSUE TITLE
Display WCAG success criterion ID

### ISSUE THREAD

**Issue by prasse** [squizlabs/HTML_CodeSniffer#142]
Really like the CS bookmarklet on Chrome, but would like to see the ID of the specific WCAG 2.0 success criterion failed instead of just the Principle and Techniques when generating a report.

**Comment by lwright-sq** [comment]
Hi there,

Sorry for the lack of initial response (besides triaging) way back when. This wasn't a high priority as you would have gathered, however I agree that the Success Criterion that's appropriate is a lot more handy than just the principle - people talk about "SC 1.3.1" but rarely talk about "Principle 1", for instance.

The trunk version has replaced the Principle with the Success Criterion applicable. Due to space constraints, the Principle has been removed, but since the SC will link to the appropriate spot in the main standard document, you can get your bearings from there.

Thanks for the report.

**Comment by prasse** [comment]
Luke – Thanks. I’m not a system analyst, so I don’t know how to recompile the bookmarklet so it includes the specific files you changed.
Sorry about that ignorance.
So do you know when the bookmarklet will be updated to show SCs?

Thanks again!

- Mike

**Comment by ironikart** [comment]
> So do you know when the bookmarklet will be updated to show SCs?

We don't have a release date scheduled, but will look at doing it when we get a chance to properly test the most recent round of changes and fixes. Expect it in the near future. In the meantime if you really need it you could follow the instructions in the project's README for generating a development version of HTML_CodeSniffer.
